go/vt/tabletserver/config.go

```diff
@@ -154,7 +154,7 @@ var DefaultQsConfig = Config{
 	SpotCheckRatio:       0,
 	StrictMode:           true,
 	StrictTableAcl:       false,
-	TerseErrors:          false,
+	TerseErrors:          true,
```

Comment:
We want this to be false by default. We don't want to encourage the use of terse errors.